src/engine/Operation.cpp

```diff
@@ -270,10 +378,12 @@ void Operation::updateRuntimeInformationOnSuccess(
 
 // ____________________________________________________________________________________________________________________
 void Operation::updateRuntimeInformationOnSuccess(
-    const ConcurrentLruCache ::ResultAndCacheStatus& resultAndCacheStatus,
+    const ConcurrentLruCache::ResultAndCacheStatus& resultAndCacheStatus,
     Milliseconds duration) {
+  const auto& result = resultAndCacheStatus._resultPointer->resultTable();
   updateRuntimeInformationOnSuccess(
-      *resultAndCacheStatus._resultPointer->resultTable(),
+      // TODO<RobinTF> find a better representation for "unknown" than 0.
```

Comment:
This concept doesn't 100% make sense for lazy evaluation. We could of course provide a lower bound for qlever-ui to display by adding this to the onSizeChanged listener or something